### DISCUSSION TITLE
Removing or changing window border color for telescope select

### DISCUSSION THREAD

**Discussion by mystilleef** [stevearc/dressing.nvim#62] [Q&A] [answered]
![Screenshot from 2022-09-07 13-16-20](https://user-images.githubusercontent.com/273399/188940323-61eb9f40-a0ab-4af8-a515-b8a67b09e1b0.png)

How do I remove the cyan colored border from this window? What highlight group do I need to adjust? This is code action using the `Telescope` backend.

**Comment by stevearc** [comment]
Since this is a question about the telescope UI it would be more appropriate to ask it at [telescope.nvim](https://github.com/nvim-telescope/telescope.nvim), but that is probably the built-in `FloatBorder` highlight group.

**Reply by mystilleef** [reply]
The default `Telescope` pickers don't show the border color. That's why I'm asking here. I've tried changing `FloatBorder` but that has no effect on it for some reason. The code action window doesn't respect my `Telescope*Border` highlights, either. This is what my usual `Telescope` window looks like.

![Screenshot from 2022-09-07 22-00-56](https://user-images.githubusercontent.com/273399/189017036-1ec779ab-006f-4dc4-988b-43e57859ddaf.png)

**Reply by stevearc** [reply]
dressing uses the [dropdown theme by default](https://github.com/stevearc/dressing.nvim/blob/b188b7750c78c0dbe0c61d79d824673a53ff82db/lua/dressing/select/telescope.lua#L82), which is why it looks different. If you do `:lua require'telescope.builtin'.find_files(require('telescope.themes').get_dropdown({}))`, what do you see?

**Reply by mystilleef** [reply]
Yeah, I use the default theme, `dropdown`. I just have a borderless setup, which is why I'm confused as to why borders are still showing up in the code action window. 

This is what I get with the command. For some reason, the code action window is not respecting my `Telescope*Border` highlights.

![Screenshot from 2022-09-07 23-18-03](https://user-images.githubusercontent.com/273399/189026810-dce6a53c-ebe5-4d06-807f-3a0bc9db5b81.png)

**Comment by mystilleef** [comment]
Ah, I think it's might have to do with `borderchars`. My telescope setting removes `borderchars`. So, I guess, I have to pass `borderchars` to the `telescope` key in `select`?

**Comment by mystilleef** [comment, marked as answer]
Okay, figured it out. I had to pass `borderchars` to telescope.

```lua
    use({
      "stevearc/dressing.nvim",
      config = function()
        require("dressing").setup({
          select = {
            telescope = {
              borderchars = { " ", " ", " ", " ", " ", " ", " ", " " },
            },
          },
        })
      end,
    })
```

I had assumed `select` inherits `Telescope's` settings, but it doesn't. Thankfully, there's a way to pass in the setting you care about.

Thanks for your help.
